src/components/settings/SettingsSSOTab/SSOStepper.jsx

```diff
@@ -121,14 +132,19 @@ const SSOStepper = ({ enterpriseSlug, enterpriseId, enterpriseName }) => {
             )}
             <Stepper.ActionRow.Spacer />
             <Button
-              disabled={!isIdpStepComplete}
+              disabled={!isIdpStepComplete || idpNextButtonDisabled}
```

Comment:
I noticed that repeated clicks would make repeated calls with the same payload and since the ID updates after each successful update, the second and any future request would fail with a 404. Plus also just not great to make lots of API calls